services/commands/command_definitions/schemas.go

```diff
@@ -113,12 +113,23 @@ var argumentSchema = map[string]*schema.Schema{
 		MinItems: 1,
 		MaxItems: 1,
 		Elem: &schema.Resource{
-			Schema: general_objects.DefinitionAttributeSchema([]string{"BINARY"}, nil),
+			Schema: general_objects.DefinitionAttributeSchema(
+				[]string{"STRUCTURE", "GEOPOINT", "TLE"}, // attribute types not allowed in command definition attributes
+				nil,                                      // All fields are used
+			),
 		},
 	},
 }
 
 var dataSourceFilterSchema = map[string]*schema.Schema{
```

Comment:
created_by is missing?

Reply:
yes good catch. I will add it in the next commit.